src/backend/distributed/metadata/dependency.c

```diff
@@ -18,11 +18,13 @@
 #include "catalog/indexing.h"
 #include "catalog/pg_class.h"
 #include "catalog/pg_depend.h"
+#include "catalog/pg_shdepend.h"
 #include "catalog/pg_type.h"
 #include "distributed/commands/utility_hook.h"
 #include "distributed/listutils.h"
 #include "distributed/metadata/dependency.h"
 #include "distributed/metadata/distobject.h"
+#include "miscadmin.h"
```

Comment:
problem with this is that `"postgres.h"` needs to generally be imported before anything else. If we want to start ordering `#include` marco's (which we should) it will result in `"miscadmin.h"` to be included before `"postgres.h"`.

If we want to have `"miscadmin.h"` at the top it would be better suited in its own group to prevent the ordering issue.